src/assets/scss/base.scss

```diff
@@ -113,3 +113,350 @@ html {
 .cursor-default {
   cursor: default; /* arrow */
 }
+
+.bg-grey-1 {
+  background-color: $gray1;
+}
+
+.bg-grey-2 {
+  background-color: $gray2;
+}
+
+.bg-grey-3 {
+  background-color: $gray3;
+}
+
+.bg-grey-4 {
+  background-color: $gray4;
+}
+
+.bg-grey-5 {
+  background-color: $gray5;
+}
+
+.colour-link {
+  color: $app-blue;
+}
+
+.colour-p-blue-text {
+  color: $non-clickable-blue !important;
+}
+
+.colour-text {
+  color: $text !important;
+}
+
+.colour-dk-text {
+  color: $dk-text !important;
+}
+
+.colour-white {
+  color: white !important;
```

Comment:
This one doesn't look very useful (since it's assigning a colour by name, not variable, which can be scoped instead of global).

As it's used in only 1 place, can you please move this to where it's used?